From 58d1042bc55ad51757a5be8677f2d71b773fb502 Mon Sep 17 00:00:00 2001
From: Sam Hewitt <sam@snwh.org>
Date: Wed, 18 Sep 2019 14:48:06 -0400
Subject: [PATCH 23/27] librem5: updates page: fix list icon

---
 src/gs-app-row.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gs-app-row.c b/src/gs-app-row.c
index 262e1ee9..3d789f1b 100644
--- a/src/gs-app-row.c
+++ b/src/gs-app-row.c
@@ -171,7 +171,7 @@ gs_app_row_refresh_button (GsAppRow *app_row, gboolean missing_search_result)
 			 * allows the application to be easily removed */
 			gtk_stack_set_visible_child (GTK_STACK (priv->button_stack), priv->button_image);
 			gtk_image_set_from_icon_name (GTK_IMAGE (priv->button_image),
-						      "list-remove-symbolic",
+						      "edit-delete-symbolic",
 						      GTK_ICON_SIZE_BUTTON);
 		}
 		break;
@@ -183,7 +183,7 @@ gs_app_row_refresh_button (GsAppRow *app_row, gboolean missing_search_result)
 		 * allows the application to be easily removed */
 		gtk_stack_set_visible_child (GTK_STACK (priv->button_stack), priv->button_image);
 		gtk_image_set_from_icon_name (GTK_IMAGE (priv->button_image),
-					      "list-remove-symbolic",
+					      "edit-delete-symbolic",
 					      GTK_ICON_SIZE_BUTTON);
 		break;
 	case AS_APP_STATE_INSTALLING:
-- 
2.26.2

