From: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Date: Tue, 27 Sep 2022 09:42:31 -0300
Subject: [PATCH] Disable offline updates

---
 plugins/packagekit/gs-plugin-packagekit.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/plugins/packagekit/gs-plugin-packagekit.c b/plugins/packagekit/gs-plugin-packagekit.c
index da8257a..2aaa4cc 100644
--- a/plugins/packagekit/gs-plugin-packagekit.c
+++ b/plugins/packagekit/gs-plugin-packagekit.c
@@ -740,6 +740,8 @@ gs_plugin_packagekit_add_updates (GsPlugin *plugin,
 				  GCancellable *cancellable,
 				  GError **error)
 {
+	return TRUE; //using unattended-upgrades in Debian and Ubuntu
+
 	g_autoptr(GsPackagekitHelper) helper = gs_packagekit_helper_new (plugin);
 	g_autoptr(PkTask) task_updates = NULL;
 	g_autoptr(PkResults) results = NULL;
@@ -3779,6 +3781,8 @@ gs_plugin_packagekit_download_async (GsPluginPackagekit  *self,
                                      GAsyncReadyCallback  callback,
                                      gpointer             user_data)
 {
+	return TRUE; //using unattended-upgrades in Debian and Ubuntu
+
 	GsPlugin *plugin = GS_PLUGIN (self);
 	g_autoptr(GTask) task = NULL;
 	g_autoptr(DownloadData) data_owned = NULL;
@@ -4076,6 +4080,8 @@ update_apps_download_cb (GObject      *source_object,
 	if (!(data->flags & GS_PLUGIN_UPDATE_APPS_FLAGS_NO_APPLY)) {
 		gboolean trigger_update = FALSE;
 
+		return TRUE; //using unattended-upgrades in Debian and Ubuntu
+
 		/* Are any of these apps from PackageKit, and suitable for offline
 		 * updates? If any of them can be processed offline, trigger an offline
 		 * update. If all of them are updatable online, don’t. */
